### DISCUSSION TITLE
Is there any support for failover in pymssql?

### DISCUSSION THREAD

**Discussion by chriss1245** [pymssql/pymssql#867] [Q&A]
We switched some time ago to pymssql from pyodbc. After some time we had an issue in our database. So we now want to add a failover partner so that we can keep connected.  I have been searching about and it seems that pymssql does not support failover.

**Comment by termim** [comment]
What do you mean by failover support?

**Reply by chriss1245** [reply]
Sorry, I am not quite familiarized with this. I wonder if there is a failoverPartner option in which I can specify the IP address of a mirror database.  So that  I can connect in case the main database is down without touching the connection string.

**Comment by termim** [comment]
There is nothing in pymssql that can do this automatically. It should be done in your application.

**Reply by chriss1245** [reply]
I see... thanks
